powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/SignatureAuditRequest.java

```diff
@@ -30,9 +32,17 @@
 @Data
 public class SignatureAuditRequest {
 
+    @Schema(description = "The identifier of the user")
+    @NotBlank(message = "User ID must not be empty when requesting signature audit")
     private String userId;
+
+    @Schema(description = "The identifier of the application")
     private String applicationId;
+
+    @Schema(description = "The timestamp from parameter in date filter")
```

Comment:
Suggestion to use different wording
```suggestion
    @Schema(description = "The starting timestamp for filtering activation history")
```